src/plugins/pluginManager/plugin.ts

```diff
@@ -22,6 +22,7 @@ import {
   ProposalPlugin,
   transactionResultHandler
 } from '../../index'
+const abiDecoder = require('abi-decoder')
```

Comment:
please add a comment that we use `abi-decoder` instead of ethers due to some unexplained issue with ethers interface.functions.FUNCTIONNAME.decode().
This might be solved when shifting to ethers 5.0 .